Add Success Criteria to quickstart guide (#426)
oddbird/oddleventy: content/quickstart.md

Author: I plan to update the text wrapping to match the rest of the file once the actual contents are reviewed

Author: Update in 2ea8cd85b9579f2de9e0a00e60ad8d6cba11d9be

[harvest]: https://www.getharvest.com/
[accounts]: mailto:accounts@oddbird.net

> ## Success Criteria:

Author: Not sure if the very end of the guide is the best place to put the section, open to suggestions

Reviewer: I'd recommend after "Participation" and before "Finding Clients".

Author: After reading it again in context of the rest of the guide, I think it makes sense to have it at the end because it touches on several points introduced earlier, but I'm still open to moving it

Reviewer 2: I think the end would be fine.

This guide will help you understand the expectations for your role at OddBird.
The criteria are divided into three groups: Work Quality, Communication, and
Learning and Growth. Each group has three levels of performance: Under performs,

> Meets, and Exceeds. You can use this guide to assess your own performance,

Reviewer 2: ```suggestion
Meets Expectations, and Exceeds Expectations. You can use this guide to assess your own performance,
```

identify your strengths and areas for improvement, and set your goals and action

> plans.

Author: I came up with these three arbitrary groups, what do you think? Also, is the current "list" style more appropriate than a table? Or plain prose?

Reviewer 2: These groups make sense to me! I really like the list style and find it easy to read/comprehend quickly.

### Work Quality

> #### Under performs

Reviewer 2: ```suggestion
#### Under Performs
```

- Lacks initiative and waits for directions or assignments on high priority work
- Produces low-quality or incomplete work that requires excessive rework or
  refactoring from others
- Uses outdated or inappropriate tools, technologies, or methodologies
- Avoids or gives up on challenges

> - Does not plan or dissect big assignments into smaller tasks

Reviewer 2: ```suggestion
- Does not plan ahead or break big assignments into smaller tasks
```

Reviewer 3: I think "plan ahead" has a slightly different meaning than the intended one, and I personally prefer "dissect" rather than "break" here. 🤷 Maybe there's a synonym we can find that would be clearer than "plan"?

Reviewer: Would "Does not create plans for big assignments, or dissect them into smaller tasks" work?

Reviewer 2: I really like James' wording suggestion here. This can be applied to all the 3 sections where it comes up.

Author: Fixed in 9c8418ba1f92581a5310d6aa36bddf39c5f7d104

> #### Meets

Reviewer 2: ```suggestion
#### Meets Expectations
```

- Takes initiative on high priority work by chipping away at it

> - Delivers complete and high-quality work and acts on feedback from others

Reviewer 2: ```suggestion
- Delivers complete and high-quality work, and acts on feedback from others
```

> - Approaches challenges by spending time planning and dissecting out assignments

Reviewer 2: ```suggestion
- Approaches challenges by spending time planning and breaking big assignments into smaller tasks
```

> #### Exceeds

Reviewer 2: ```suggestion
#### Exceeds Expectations
```

> - Takes initiative planning and breaking down upcoming work

Reviewer 2: ```suggestion
- Takes the initiative and leads others in planning and breaking down upcoming work
```
I recommend moving this bullet point to the bottom of this list since the analogous point in the other 2 sections is the last one in those lists. Also this one seems quite similar to the analogous bullet point under "Meets," so I added a bit more.

- Takes ownership of features and infrastructure
- Leads internal review processes and delivers high-quality, actionable feedback
  to others
- Makes requests for input early in process to tackle challenging work

### Communication

> #### Under performs

Reviewer 2: ```suggestion
#### Under Performs
```

> - Rarely communicates or does it in an untimely manner

Reviewer 2: ```suggestion
- Rarely communicates or does so in an untimely manner
```

> - Misses a large part or entire meetings without notice

Reviewer 2: ```suggestion
- Misses large parts of or entire meetings without notice
```

> - Does not document their break down of upcoming work or status of in-progress

Reviewer 2: ```suggestion
- Does not document the breakdown of their upcoming work or the status of their in-progress
```

  work, and does not reflect on their contributions, accomplishments, or lessons
  from challenges (self-retro)
- Does not ask for help or guidance when blocked, and does not participate or
  learn from review and discussions
- Shows disinterest or indifference when planning new work

> #### Meets

Reviewer 2: ```suggestion
#### Meets Expectations
```

- Communicates frequently and in time

> - Attends most meetings and notifies in advance when they cannot attend

Reviewer 2: ```suggestion
- Attends most meetings and notifies team in advance when they cannot attend or will be late
```

- Posts a breakdown of their daily assignments documenting completed work, the
  status of in-progress work, and upcoming tasks
- Documents their contributions, accomplishments, and lessons from challenges
  (self-retro)
- Reaches out with questions when blocked and engages with review and
  planning by asking more questions

> #### Exceeds

Reviewer 2: ```suggestion
#### Exceeds Expectations
```

- Communicates with trust in advance and requests engagement from others

> - Leads meetings and discussions on internal and external projects

Reviewer 2: ```suggestion
- Leads meetings and discussions about internal and external projects
```

- Follows up during and after reviews and discussions with questions that help
  confirm understanding
- Generates internal documents, blog posts, and other valuable content based on
  their contributions (especially accomplishments and challenges)

### Learning and Growth

> #### Under performs

Reviewer 2: ```suggestion
#### Under Performs
```

>   their contributions (especially accomplishments and challenges)

Reviewer: Perhaps covered in other points throughout, but another area of exceeding expectations would be challenging and defining process, and not just following existing process. Perhaps something like-

Meets Expectations-
`Challenges the team processes and looks for ways to improve them`

Exceeds Expectations-
`Adjusts the team processes, listening to feedback and guiding the team through the changes`

These are mostly pulled from http://www.engineeringladders.com/

Author: Great link! I added both and another entry to "Underperform" in baec15edfa062d3ac7c7ea22c86a4f0cf645f8bf

- Does not participate or learn from review and discussions, and asks few or
  no questions
- Resists or rejects change and feedback, and does not adjust to the changing
  needs and expectations of projects
- Lacks initiative and collaboration skills, and does not support the team's
  goals and objectives

> #### Meets

Reviewer 2: ```suggestion
#### Meets Expectations
```

- Engages with review and discussions by asking more questions
- Reaches out with questions when blocked
- Acts on feedback as an opportunity to learn and improve, remains flexible and
  adaptable to change

> #### Exceeds

Reviewer 2: ```suggestion
#### Exceeds Expectations
```

> - Demonstrates curiosity and pursues challenges that help them and the team grow

Reviewer 4: I think the rest of the quickstart was written in second person. The change didn't stand out to me until this use of `them`. Not sure it matters much.

Author: Switched to second person in 84dd72f9ebf644d93efa4e1b16a37a27571a5ae9
